src/Source/OrientedImageSource.js

```diff
@@ -0,0 +1,43 @@
+import Source from 'Source/Source';
+
+/**
+ * @classdesc OrientedImageSource is a specific source used to load oriented images.
+ * @extends Source
+ */
+class OrientedImageSource extends Source {
+    /**
+     * @constructor
+     * @param { Object } source - Configuration object
+     * @param { string } source.url - Url for all the textures.
+     * This Url must contains {sensorId} and {cameraId}, and these pattern will be replaced be the {@link OrientedImageProvider} o find the good texture for each camera for each panoramic.
+     */
+    constructor(source) {
+        source.format = source.format || 'json';
+        super(source);
+        this.protocol = 'orientedImage';
+    }
+
+    /**
+     * Build the url from extent.
+     *
+     * @param      {OrientedImageCamera}  camera  The camera
+     * @param      {Object}  pano    The panoramic object
+     * @return     {string}  Url of the image
+     */
+    urlFromExtent(camera, pano) {
+        return this.imageUrl(camera.name, pano.id);
+    }
+
+    /**
+     * Build the url of the image, for a given panoramic id, and a given camera id.
+     *
+     * @param      {string}  cameraId  Id of the camera
+     * @param      {string}  panoId   Id of the panoramic
+     * @return     {string}  Url of the image
+     */
+    imageUrl(cameraId, panoId) {
```

Comment:
Is there a point to have this method ? If not, please build the entire url in `urlFromExtent`.

Reply:
Yes, it's the most simple string replacing method, the heart of this source, and it doesn't change when we change the way the source is called.. So I keep it.